README.md

```diff
@@ -42,9 +49,10 @@ Cloud Controller currently supports [webdav](http://www.webdav.org/) and the fol
 
 ### Runtime
 
-The Cloud Controller uses [Diego](https://github.com/cloudfoundry/diego-release) to stage and run apps and tasks.
+The Cloud Controller on VMs uses [Diego](https://github.com/cloudfoundry/diego-release) to stage and run apps and tasks. See [Diego Design Notes](https://github.com/cloudfoundry/diego-design-notes) for more details.
+
+When deployed on kubernetes, Cloud Controller uses [Eirini](https://github.com/cloudfoundry-incubator/eirini) to run apps directly on the kubernetes cluster.
```

Comment:
I wonder if this should mention it uses kpack to stage apps using Cloud Native Buildpacks?

Also a nit, but I think Kubernetes should be capitalized.